testutil/node.go

```diff
@@ -106,3 +107,26 @@ func (t *TestNode) handleMessages() {
 		}
 	}
 }
+
+// TimeoutOnRound advances time until the node times out of the given round.
+func (t *TestNode) TimeoutOnRound(round uint64) {
+	startTime := t.E.StartTime
```

Comment:
I think it makes sense to have a time as part of the `TestNode` struct, otherwise you can only use this method properly once. When this method returns, the start time of the node is out of sync with the `E.StartTime` and you will start from the old start time the next time you invoke this method. Instead I think we should store the current time in the `TestNode` and when you update the time you also save it in the `TestNode`. This way, next time you invoke this method, you will fetch the updated start time.